spec/spec_helper.rb

```diff
@@ -5,6 +5,9 @@
 require_relative 'support/database_cleaner'
 require_relative '../app.rb'
 
+require 'valid_attribute'
+require 'shoulda/matchers'
+
```

Comment:
Could you move these `require` statements up with the other `require` statements?

```
# requires

# require_relatives
```